lib/clever-ruby/models/school.rb

```diff
@@ -192,26 +192,6 @@ def valid?
       return true
     end
 
-    # Custom attribute writer method checking allowed values (enum).
-    # @param [Object] high_grade Object to be assigned
-    def high_grade=(high_grade)
-      validator = EnumAttributeValidator.new('String', ["1", "2", "3", "4", "5", "6", "7", "8", "9", "10", "11", "12", "PreKindergarten", "Kindergarten", "PostGraduate", "Other"])
```

Comment:
does it make sense to do it as a part of configuration?
https://github.com/ClassTagInc/clever-ruby/blob/master/lib/clever-ruby/configuration.rb

Reply:
I don't see a lot of sense in it, because looks like we'll need to implement own api layer for Clever (like for ClassLink), because right now gem produces too many bottlenecks and has very poor error handling which is not suitable for us. Also tons of validations which don't pass their own data through the data models in gem. Painful.